docs/docs/rest-api/public/api/v2/schema/AppDefinition.json

```diff
@@ -1,712 +0,0 @@
-{
```

Comment:
Why would we want to support both at the same time? As part of some "deprecation" period for the schema?